config-example/el/genesis-config.yaml

```diff
@@ -24,7 +24,7 @@ el_premine:
   "m/44'/60'/0'/0/18": 1000000000ETH
   "m/44'/60'/0'/0/19": 1000000000ETH
   "m/44'/60'/0'/0/20": 1000000000ETH
-el_premine_addrs: {}
+additional_preloaded_contracts: ${ADDITIONAL_PRELOADED_CONTRACTS}
```

Comment:
I'd suggest keeping the old property (el_premine_addrs) untouched here.
It's in use by ephemery and probably other projects too :)